packages/docs/docker.package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "docs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "npm run generate:components:docs && next dev",
+    "build": "npm run generate:components:docs && next build",
+    "start": "next start -H 0.0.0.0 -p ${PORT:-3000}",
+    "generate:components": "node docgen.js",
+    "generate:components:docs": "npm run generate:components && node docs-generator.js",
+    "generate:watch": "concurrently \"npm run generate:components -- --watch\" \"npm run generate:components:docs -- --watch\"",
+    "serve": "node server.js"
+  },
+  "dependencies": {
+    "@patternfly/react-core": "^4.79.2",
+    "@patternfly/react-icons": "^4.7.18",
+    "@patternfly/react-table": "^4.19.24",
+    "@redhat-cloud-services/frontend-components": "^3.0.10",
+    "classnames": "^2.2.6",
+    "next": "^10.0.7",
+    "prism-react-renderer": "^1.1.1",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-json-view": "^1.19.1",
+    "react-redux": "^7.2.2",
+    "react-router-dom": "^5.2.0",
+    "redux": "^4.0.5"
+  },
+  "devDependencies": {
+    "@mdx-js/loader": "^1.6.22",
+    "@next/mdx": "^10.0.5",
+    "@zeit/next-css": "^1.0.1",
+    "codesandbox": "^2.2.1",
+    "concurrently": "^5.3.0",
+    "file-loader": "^6.2.0",
+    "jsdoc": "^3.6.6",
+    "jsdoc3-parser": "^2.0.0",
+    "next-transpile-modules": "^6.0.0",
+    "raw-loader": "^4.0.2",
+    "react-docgen": "^5.3.1",
+    "react-jss": "^10.5.0",
+    "url-loader": "^4.1.1",
+    "webpack": "^4.0.0"
```

Comment:
Do you need to specify webpack? Isn't it a part of Next?

Reply:
Yes, we do. There is an issue with webpack 5 and one of the CSS decorators. I don't know what is it exactly so we need webpack 4 to be installed.